src/components/Microsoft.DotNet.UpgradeAssistant.MSBuild/MSBuildProject.File.cs

```diff
@@ -38,26 +38,13 @@ public ProjectRootElement ProjectRoot
             }
         }
 
-        public string Sdk
-        {
-            get
-            {
-                var sdk = ProjectRoot.Sdk;
-
-                if (sdk is null)
-                {
-                    throw new ArgumentOutOfRangeException("Should check IsSdk property first");
-                }
-
-                return sdk;
-            }
-        }
-
         public bool IsSdk =>
```

Comment:
This should be Sdk.Any() as this is a check if we're using Sdk style. Not sure why it was checking for specific things below.